createdb.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/sh
+# TODO - it'll be trivial, but put this into a helper perhaps config.go and
+# call it any time we try to open the db and find it doesn't exist.
+# For now, run this script by hand to create the db
+db=/var/lib/lxd/lxd.db
+if [ $# -eq 1 ]; then
+	db=$1
+fi
+rm -f $db
+cat << EOF | sqlite3 $db
+create table certificates (
+id integer primary key autoincrement not null,
+fingerprint varchar(255) not null,
+type integer,
+name varchar(255) not null,
+certificate blob not null,
+unique (fingerprint)
+);
+create table containers (
+id integer primary key autoincrement not null,
+name varchar(255) not null,
+architecture integer,
```

Comment:
Quoting St�phane Graber (notifications@github.com):
> > +	db=$1
> > +fi
> > +rm -f $db
> > +cat << EOF | sqlite3 $db
> > +create table certificates (
> > +id integer primary key autoincrement not null,
> > +fingerprint varchar(255) not null,
> > +type integer,
> > +name varchar(255) not null,
> > +certificate blob not null,
> > +unique (fingerprint)
> > +);
> > +create table containers (
> > +id integer primary key autoincrement not null,
> > +name varchar(255) not null,
> > +architecture integer,
> 
> Should be NOT NULL

Did this and all the others which don't match the spec regarding NOT NULL